src/pages/PvPartners.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import '../static/css/PvPartners.css';
+import kia from '../static/images/kia-logo-png-transparent 1.png';
+import gdot from '../static/images/DOT-GA-US-Logo-2018 1.png';
+import facebook from '../static/images/facebook.png';
+import twitter from '../static/images/twitter.png';
+import linkedin from '../static/images/linked in.png';
+import instagram from '../static/images/instagram.png';
+import raylogo from '../static/images/ray-anderson-logo 1.png';
+import hannahlogo from '../static/images/hannahlogo.png';
+
+class PvPartners extends React.PureComponent {
+  render() {
+    return (
+      <div className="partners-background">
+        <div className="partners-body">
+          <img className="kia" src={kia} alt="Kia logo" />
+          <img className="gdot" src={gdot} alt="Dot logo" />
+          <img className="ray" src={raylogo} alt="Ray C. Anderson logo" />
+          <img className="hannah" src={hannahlogo} alt="Hannah logo" />
+          <a href="https://www.facebook.com/TheRayHighway/">
+            <img className="facebook" src={facebook} alt="facebook logo" />
+          </a>
+          <a href="https://twitter.com/therayhighway?lang=en">
```

Comment:
Rather than reimplementing the same page as the Partner page, you could give the Partner component some properties that can be toggled on or off depending on whether its PV4EV or Wheelright. Could get rid of this page and extend the old page at `src/pages/Partners.js`